seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/excel/ExcelCellUtils.java

```
@@ -0,0 +1,185 @@
+
+import com.alibaba.excel.enums.CellDataTypeEnum;
+import com.alibaba.excel.metadata.Cell;
+import com.alibaba.excel.metadata.data.ReadCellData;
+import lombok.SneakyThrows;
+
+import javax.annotation.Nullable;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class ExcelCellUtils implements Serializable {
+
+    static final long serialVersionUID = 42L;
+
+    private DateTimeFormatter dateFormatter;
+    private DateTimeFormatter dateTimeFormatter;
+    private DateTimeFormatter timeFormatter;
+
+    protected Config pluginConfig;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public ExcelCellUtils(
+            Config pluginConfig,
+            String dateFormatterPattern,
+            String dateTimeFormatterPattern,
+            String timeFormatterPattern) {
+        this.pluginConfig = pluginConfig;
+        this.dateFormatter = DateTimeFormatter.ofPattern(dateFormatterPattern);
+        this.dateTimeFormatter = DateTimeFormatter.ofPattern(dateTimeFormatterPattern);
+        this.timeFormatter = DateTimeFormatter.ofPattern(timeFormatterPattern);
+    }
+
+    private String getCellValue(ReadCellData cellData) {
+
+        if (cellData.getStringValue() != null) {
+            return cellData.getStringValue();
+        } else if (cellData.getNumberValue() != null) {
+            return cellData.getNumberValue().toString();
+        } else if (cellData.getOriginalNumberValue() != null) {
+            return cellData.getOriginalNumberValue().toString();
+        } else if (cellData.getBooleanValue() != null) {
+            return cellData.getBooleanValue().toString();
+        } else if (cellData.getType() == CellDataTypeEnum.EMPTY) {
+            return "";
+        }
+        return null;
+    }
+
+    @SneakyThrows
```

This annotation cannot ignore all errors